internal/tests/api/with-ns-registered/minipool_test.go

```diff
@@ -420,6 +422,79 @@ func TestMinipoolDeposit(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, types.MinipoolStatus_Staking, mp.Common().Status.Formatted())
 	t.Log("Minipool is in staking")
+
+	// Fast forward time
+	slotsToAdvance = 1200 * 60 * 60 / 12
+	err = testMgr.AdvanceSlots(uint(slotsToAdvance), false)
+	require.NoError(t, err)
+	t.Logf("Advanced %d slots", slotsToAdvance)
+
+	var wethBalanceYieldDistributorBefore *big.Int
+	err = qMgr.Query(func(mc *batch.MultiCaller) error {
+		weth.BalanceOf(mc, &wethBalanceYieldDistributorBefore, csMgr.YieldDistributor.Address)
+		return nil
+	}, nil)
+	require.NoError(t, err)
+
+	// Send ETH to YieldDistributor to trigger finalizeInterval
+	// sendResponse, err := hd.Wallet.Send(eth.EthToWei(1), "eth", csMgr.YieldDistributor.Address)
+	// require.NoError(t, err)
+	// MineTx(t, sendResponse.Data.TxInfo, sendEthOpts, "Sent ETH to the YieldDistributor")
+
+	sendEthTx := eth.TransactionInfo{To: csMgr.YieldDistributor.Address, Value: eth.EthToWei(1)}
```

Comment:
Recommend using `txMgr.CreateTransactionInfoRaw()` here instead of just making it manually, which will create it and do the simulation for you (which `MineTx()` checks).

For the `opts` do this:
```
	sendOpts := &bind.TransactOpts{
		From:  deployerOpts.From,
		Value: big.NewInt(1e18),
	}
```